CTP/README.md

```diff
@@ -93,94 +109,174 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
 	  Done
 	  URL:  http://127.0.0.1:8888
 	  ```
-	* Please open the ``URL`` with your browser.
+	* Please open the ``URL`` with your browser
   
 - **SHELL**
   - Prepare
-	* Use one server as controller to checkout CTP, and test node may be one or more, they will be controlled by controller, and CTP must be deployed on each node.
-	* Controller Node configuration:
+	* Prepare at least two accounts (e.g., one account ``controller`` as controller, another account ``shell_instance1`` as test instance)
+	* Check out CTP for each account and configure environment variables for CTP ``controller`` and test instance ``shell_instance1``
 	
 	  ```
-	    Test nodes are configured in CTP/conf/shell.conf as below:
-		env.instance1.ssh.host=192.168.1.10
-		env.instance1.ssh.port=22
-		env.instance1.ssh.user=<user>
-		env.instance1.ssh.pwd=<pwd>
-		env.instance1.cubrid.cubrid_port_id=11523
-		env.instance1.broker1.BROKER_PORT=35000
-		env.instance1.broker2.BROKER_PORT=35500
-		
-		env.instance2.ssh.host=192.168.1.11
-		env.instance2.ssh.port=22
-		env.instance2.ssh.user=<user>
-		env.instance2.ssh.pwd=<pwd>
-		env.instance2.cubrid.cubrid_port_id=11523
-		env.instance2.broker1.BROKER_PORT=35000
-		env.instance2.broker2.BROKER_PORT=35500
-		
-	  ```
-	  
-	  Regarding more parameters for shell testing, please refer to [CTP/conf/shell.conf](conf/shell.conf)
-	* Test Node:
-	  
-	  ```
-	     export JAVA_HOME=$java_installation_directory 
-	     export CTP_HOME=$CTP_installation_directory
+	   // on controller node
+	   JAVA_HOME (e.g., export JAVA_HOME=$HOME/opt/jdk1.6.0_07)
+
+	   // on test instance node
+	   CTP_HOME (e.g., export CTP_HOME=$HOME/CTP) 
+	   JAVA_HOME (e.g., export JAVA_HOME=$HOME/opt/jdk1.6.0_07)
 	  ```
+	* **Example** ``shell.conf`` for test instance, scenario and test build:
+	
+	   ```
+	   # Test instance information:
+	   env.instance1.ssh.host=192.168.1.10
+	   env.instance1.ssh.port=22
+	   env.instance1.ssh.user=shell_instance1
+	   env.instance1.ssh.pwd=123456
+	   env.instance1.cubrid.cubrid_port_id=11523
+	   env.instance1.broker1.BROKER_PORT=35000
+	   env.instance1.broker2.BROKER_PORT=35500
+	   ```
+	   ```
+	   # Define the path of test cases used for testing, it should be checked out on test node
+	   main.testcase.root=/home/qa/shell_instance1/shell

```

Comment:
Why don't you use the 'scenario' paremeter name for this? and use a $HOME var? like below:
->	   scenario=${HOME}/cubrid-testcases/shell